src/lib/daemon_rpcs/daemon_rpcs.ml

```diff
@@ -86,6 +87,7 @@ module Types = struct
             ("User_commands Sent", Int.to_string (f x)) :: acc )
           ~run_snark_worker:(fun acc x ->
             ("Snark Worker Running", Bool.to_string (f x)) :: acc )
+          ~proposal_interval:(fun acc x -> ("Proposal Interval", f x) :: acc)
```

Comment:
This should just pull the value from `Consensus.Mechanism.block_interval_ms` instead. "CODA_PROPOSAL_INTERVAL" is the config var specifically for proof of signature, and this PR also decouples the default parameter displayed from the default set in consensus.

Reply:
@nholland94 cool - didn't know I could pull it from there.  will try.

Reply:
@nholland94 using Consensus.mechanism